packages/zero-unplugin/src/index.ts

```diff
@@ -264,16 +268,16 @@ export const plugin = createUnplugin<PluginOptions, true>((options) => {
                 // this file should exist in the package
                 id.endsWith('@mui/zero-runtime/styles.css') ||
                 id.endsWith('/zero-runtime/styles.css') ||
-                id.endsWith('@mui/zero-runtime/theme') ||
-                id.endsWith('/zero-runtime/theme.js')
```

Comment:
What's the problem with these lines?

Reply:
Earlier, it was just a theme.js file but later on, I moved it into a `theme` directory. So the resolved path became `theme/index.js` resulting in not generating the source.